docker/subset-fastq-dataset/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./prepSubset.sh \
+    -a gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/Optimus/4a43c049-39bc-45be-a7a4-fac7e4dc6a41/call-StarAlign \
```

Comment:
do your cromwell execution buckets have a TTL on them?

Reply:
But, we actually don't need this file

Reply:
Do you mean the file in the broad-dsde-mint-dev-cromwell-execution bucket or this run.sh file?